src/protocol/payload/tx.rs

```diff
@@ -149,7 +149,7 @@ pub struct TxV2 {
 
     // Only present if the join_split count > 0.
     join_split_pub_key: Option<[u8; 32]>,
-    join_split_sig: Option<[u8; 32]>,
+    join_split_sig: Option<[u8; 64]>,
```

Comment:
we could create a Sig/PubKey type or a struct to avoid repeating these array sizes, but no need for this code now

Reply:
There are many more fields that could be wrapped into some struct/type. 
However to look consistently it would be good to refactor all the fields.